brave/brave-core
# Remove support for creator-set tip defaults

Resolves https://github.com/brave/brave-browser/issues/24548

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Enable Rewards on staging
- Visit http://laurenwags.github.io/
- Send a tip
- Verify that the tip dialog shows the standard default tip amounts and not the creator-set defaults
